src/style.css

```
@@ -0,0 +1,124 @@
+/* #region Normalize */
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+
+  --color-button-link: #00acdc;
+}
+/* #endregion */
+
+/* #region Text */
+.some-text {
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.large-text {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+/* #endregion */
+
+/* #region header */
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 14px;
+  width: 200px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-top: 15px;
+}
+
+.card__desktop {
+  margin-top: 45px;
+}
+
+.card__element {
+  margin-top: 15px;
+}
+
+.desktop {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+.desktop__title {
+  margin-bottom: 6px;
+}
+
+.element {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  margin: 20px 0;
+}
+/* #endregion */
+
+/* #region images-stars */
+.stars {
+  display: flex;
+  height: 15px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 15px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background: url(images/star-active.svg) no-repeat center;
+}
+/* #endregion */
+
+/* #region link */
+.button-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  border-radius: 5px;
+  background: var(--color-button-link);
+
+  width: 100%;
+  height: 40px;
+  line-height: 16px;
+}
+
+.button-link:hover {
+  border: 1px solid var(--color-button-link);
```

```suggestion
  border-color: var(--color-button-link);
```
don't change the whole border on hover, change only border-color